src/Microsoft.OpenApi.Hidi/OpenApiService.cs

```diff
@@ -52,7 +54,11 @@ string filterbycollection
             }
             catch (ArgumentNullException ex)
             {
-                logger.LogError(ex.Message);
+#if DEBUG
```

Comment:
just as a quick though, putting those conditional compilation directives all the way to the left might be because I'm a dinosaur and that's it
![GrandmaDinosaurGIF](https://user-images.githubusercontent.com/7905502/157248693-bc53871e-cfde-4978-88d6-14d2457832c8.gif)

back in 2008 those directives would not work when indented, but I see that you originally had indented versions of them that seem to be working properly

Reply:
I think even in the [docs](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives) the preprocessor directives aren't indented.

Reply:
ah, good, so I'm not the last of my kind